From 761fd2bdc1ce2bd0cd395091b9178cbf9b6f3dfb Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Fri, 18 Jan 2013 15:29:33 +0800
Subject: [PATCH 152/281] ENGR00238265 wifi:change board config for intel wifi

change to use newest wpa_supplicant_8.
Abstract special changes to a private lib which adopted by broadcom
in Android4.2.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 sabresd_6dq/BoardConfig.mk | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index 3a5a7b0..9c8a32d 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -32,8 +32,9 @@ endif
 ifeq ($(BOARD_WLAN_VENDOR),INTEL)
 BOARD_HOSTAPD_PRIVATE_LIB		 ?= private_lib_driver_cmd
 BOARD_WPA_SUPPLICANT_PRIVATE_LIB 	 ?= private_lib_driver_cmd
-WPA_SUPPLICANT_VERSION			 := VER_0_7_X_INTEL
-HOSTAPD_VERSION				 := VER_0_7_X_INTEL
+WPA_SUPPLICANT_VERSION			 := VER_0_8_X
+HOSTAPD_VERSION				 := VER_0_8_X
+BOARD_WPA_SUPPLICANT_PRIVATE_LIB         := private_lib_driver_cmd_intel
 WIFI_DRIVER_MODULE_PATH          	 := "/system/lib/modules/iwlagn.ko"
 WIFI_DRIVER_MODULE_NAME          	 := "iwlagn"
 WIFI_DRIVER_MODULE_PATH			 ?= auto
-- 
1.8.0

